expertise/service/expertise.py

```diff
@@ -177,7 +182,7 @@ def get_expertise_all_status(self, user_id, query_params):
                 self._filter_config(config)
                 result['results'].append(
                     {
-                        'job_id': job_dir,
+                        'job_id': config.job_dir,
```

Comment:
why don't you return `config` here? it seems you are making a new dictionary with the values from config plus config converted as a json, isn't that redudant?